Packages/CandidateUI/Sources/CandidateUI/HorizontalCandidateController.swift

```diff
@@ -141,20 +137,25 @@ fileprivate class HorizontalCandidateView: NSView {
             let currentWidth = elementWidths[index]
             let labelRect = NSRect(x: accuWidth, y: tooltipSize.height, width: currentWidth, height: keyLabelHeight)
             let candidateRect = NSRect(x: accuWidth, y: tooltipSize.height + keyLabelHeight + 1.0, width: currentWidth, height: candidateTextHeight)
-            (index == highlightedIndex ? darkGray : lightGray).setFill()
-            NSBezierPath.fill(labelRect)
-            (keyLabels[index] as NSString).draw(in: labelRect, withAttributes: keyLabelAttrDict)
+            var activeKeyLabelAttrDict = keyLabelAttrDict
+            if index == highlightedIndex {
+                NSColor.selectedControlColor.setFill()
+                NSBezierPath.fill(labelRect)
+                activeKeyLabelAttrDict[.foregroundColor] = NSColor.selectedControlTextColor
+            }
+            (keyLabels[index] as NSString).draw(in: labelRect, withAttributes: activeKeyLabelAttrDict)
 
             var activeCandidateAttr = candidateAttrDict
             if index == highlightedIndex {
-                NSColor.selectedTextBackgroundColor.setFill()
-                activeCandidateAttr = candidateAttrDict
-                activeCandidateAttr[.foregroundColor] = NSColor.selectedTextColor
-            } else {
-                backgroundColor.setFill()
+                if #available(macOS 10.14, *) {
+                    NSColor.controlAccentColor.setFill()
+                } else {
+                    NSColor.selectedControlColor.setFill()
+                }
+                NSBezierPath.fill(candidateRect)
+                activeCandidateAttr[.foregroundColor] = NSColor.white
```

Comment:
No action needed, just wanted to add a note here. I was wondering why this has to be `.white` instead of `.selectedControlTextColor`, and then after I tried this I found the latter could cause the text contrast to be too low in the Light theme (as selected contorl text color is very dark). This LGTM.